Lesson_3/task_3.js

```diff
@@ -44,7 +44,7 @@ const Calc = function () {
     }
 
     const runOperation = (operation_string) => {
-        full_operations = operation_string.split(' ').map(o => parseInt(o) ? parseInt(o) : o );
+        full_operations = operation_string.split(' ').map(o => o == 0 ? 0 :  (parseInt(o) || o ));
```

Comment:
Не забывайте про поддержку дробных чисел:

```suggestion
        full_operations = operation_string.split(' ').map(o => isNaN(o) ? o : parseFloat(o));
```